openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -335,12 +338,11 @@ class DeactivateLogoutView(APIView):
     -  Log the user out
     - Create a row in the retirement table for that user
     """
-    authentication_classes = (JwtAuthentication, )
+    authentication_classes = (SessionAuthentication, JwtAuthentication, )
```

Comment:
@sanfordstudent I added this so that the client could access this endpoint - without this I was getting authentication errors.

Reply:
Cool, thanks!